### DISCUSSION TITLE
Upgrade postgres to v14 get SCRAM Password error

### DISCUSSION THREAD

**Discussion by dcpc007** [requarks/wiki#4810] [Error / Bug Report]
Hi, 

I finish upgrade my postgres to "latest" version as the doc recommands (https://docs.requarks.io/install/requirements)
install via docker-compose file in version 14.

but when i try to start, i get those errors in loop :

```
db_1    | 2021-12-11 04:39:40.600 UTC [6308] FATAL:  password authentication failed for user "wikijs"
db_1    | 2021-12-11 04:39:40.600 UTC [6308] DETAIL:  User "wikijs" does not have a valid SCRAM secret.
db_1    |       Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
```

After some search, i think i can revert to the old password mechanism, but seems postgres recommand to use the new system.
I'm really not a great user of posgres. Then i prefer to ask here before trying something.

What solution to implement ?
- follow the recommandations, and have wikijs use scram password to keep the latest postgres version installed, if it's possible 
- Use latest version but try to find the configuration to revert to old password method 
- downgrade the postgres version, and to which version max ? (and then ask to adapt the install doc)

**Comment by dcpc007** [comment]
Seems no answer. And i can't look more in the code or config currently (i fear to break all too :) )

Then i find how to revert the pgsql config to use old (not recommanded) md5 password.
Working with this workaround until a real solution.

--> modifiy pg_hba.conf (last line) :
host all all all md5

**Reply by NGPixel** [reply]
The underlying pg library does support SCRAM authentication, so your postgres user is probably not configured correctly.

**Reply by dcpc007** [reply]
right, but i didn't succeed in migrate my existing account / password as i'm not a good postgres admin, only basic usage.

As it's a breaking change in the recommanded upgrades, i was trying to find something to put as procedure for everyone, but can't for now.
